### PR TITLE
Update slf4j to 1.7.19

### PR DESCRIPTION
**Pull request by ceefour**

### CODE REVIEW

File: libraries.gradle [hunk 1]
```diff
@@ -12,7 +12,7 @@ ext {
     junitVersion = '4.11'
 //    h2Version = '1.2.145'
     h2Version = '1.3.176'
-    bytemanVersion = '2.1.2'
+    bytemanVersion = '2.1.2's
```
Comment (vladmihalcea) [inline libraries.gradle]: There is a typo here (the `s` char was probably added when you started searching for slf4j).

**Comment by ceefour** [comment]
@vladmihalcea Sorry, fixed it.

**Comment by vladmihalcea** [comment]
One more thing. Hibernate uses JBoss logging, so even if we bump this version up, it will have no effect. Why do you want to have ti changed?

**Comment by ceefour** [comment]
Can't remember but I think it was a transitive dependency situation, so it's nice to have hibernate depends on latest slf4j. Since it's still 1.7.x line there's no harm in up-ing the version.

**Comment by vladmihalcea** [comment]
I saw that this dependency was removed some time ago and then added back because the metamodel generator module required it. We should investigate it this is really needed anymore.

**Comment by vladmihalcea** [comment]
I created [HHH-10781](https://hibernate.atlassian.net/browse/HHH-10781) so that we can remove all slf4j dependency since we don't use slf4j anymore.

**Pull request closed without merging**